dist/resources/validationSchema/1.0.0-rc2/flowSpecJsonSchema.json

```diff
@@ -344,6 +344,8 @@
           "type": "array"
         },
         "value": {
+          "description": "Value will accept alphanumerics, white spaces, file path, expressions, conditions,",
+          "pattern": "^[\\w \\\\\\/@:,.+*^<>=()\"'-]+$",
```

Comment:
I can't get rid of the feeling that this regex is kind of too broad and too permissive. Do you agree?
Almost to the point of not being useful haha (maybe I'm wrong). We’re basically saying that ‘all of these characters are allowed in any combination’.

Partially it will be addressed in VMO-5190 and we will have expression validation working in parallel which is great.

But regarding the regex:
- Can we distinguish between different cases: file path, expression, string, (other cases?)?
 I think there is a way to do it in regex using the pipe symbol and parenthesis. See this paragraph from the nice book ‘Eloquent JS’: https://eloquentjavascript.net/09_regexp.html#h_In3b+t6uOO.
- Under each of the cases we could include only a subset of the symbols and take their order into consideration. Say, for the file path - alphanumeric, dash, slashes, (other characters?). Or for the expressions - they should look like @() (although the expression validation will handle it better than regex anyway). Strings for text - I’m not sure what are the restrictions.

Let me know what you think 😉

Reply:
To be fair, I'm not quite experienced at regex. So maybe other developers can give better guidance.
@AlteaDown?

Reply:
The regex is not a big deal for now I think, what we need is just to provide a regex that will help us to simulate 02 scenarios: invalid data AND valid data. We can improve the regex later.

Reply:
Okay, sure